vrecord.rb

```diff
@@ -1,29 +1,32 @@
 class Vrecord < Formula
   desc "Capturing a video signal and turning it into a digital file"
   homepage "https://github.com/amiaopensource/vrecord"
-  url "https://github.com/amiaopensource/vrecord/archive/v2020-02-10.tar.gz"
-  version "2020-02-10"
-  sha256 "761aa248856eedc23619d41f5902bba8baba1c67bddc322fcaad7cb241a1edfc"
+  url "https://github.com/amiaopensource/vrecord/archive/v2020-06-26.tar.gz"
+  version "2020-06-26"
+  sha256 "e32b77f8da9ece0fe7ed2a0660fc67a1d6ef48ff8a4692161c57223cdaa3412d"
   head "https://github.com/amiaopensource/vrecord.git"
 
   bottle :unneeded
 
-  depends_on "amiaopensource/amiaos/decklinksdk"
+  # depends_on "amiaopensource/amiaos/deckcontrol"
```

Comment:
Conflict here seems fixed if merged upstream

Reply:
ya, the new deckcontrol release should fix this part on linux